### ISSUE TITLE
Auto Commenting On New Issue Creation

### ISSUE THREAD

**Issue by sejals23** [iamparas0/TIC-TAC-TOE#186]
it ensures that comments on issues follow a consistent format or structure. This helps teams maintain a standardized approach to handling issues, reducing confusion and misunderstandings.
Users don’t need to manually add repetitive information or instructions. The automatic comment can include standard guidelines, troubleshooting steps, or links to documentation.

**Comment by iamparas0** [comment]
> it ensures that comments on issues follow a consistent format or structure. This helps teams maintain a standardized approach to handling issues, reducing confusion and misunderstandings. Users don’t need to manually add repetitive information or instructions. The automatic comment can include standard guidelines, troubleshooting steps, or links to documentation.

please go for it! 
please put a short video or ss before pr

**Comment by sejals23** [comment]
@iamparas0 oK sir and thank you for the opportunity

**Comment by sejals23** [comment]
@iamparas0  Sir  I can't show the  ss or video ,once the pr is merged the feature will start working
